### ISSUE TITLE
Hoverable/clickable points don't work properly with logarithmic axis transforms

### ISSUE THREAD

**Issue by GoogleCodeExporter** [josejaner/flot#467]
```
Hovering over/clicking on points on a graph using transform/inverseTransform to 
create logarithmic axis scaling doesn't work properly.

var plot = $.plot($("#placeholder"), [ { data: 
[[1,1],[10,10],[100,100],[1000,1000],[10000,10000],[100000,100000]] } ],
                  { series: { lines: { show: true }, points: { show: true } },
                    grid: { hoverable: true },
                    yaxis: { min:1, transform: Math.log, inverseTransform: Math.exp},
                    xaxis: { min:1, transform: Math.log, inverseTransform: Math.exp }
                  });

In the example above, a the first and second points can be highlighted with the 
mouse, but the remainder can't. Setting options.grid.mouseActiveRadius=1000 
makes it possible to highlight a few more points, but the first points then 
become too sensitive.

Summary: findNearbyItem() is not consistently using pixel coordinates/axis 
transformations.
```

Original issue reported on code.google.com by `tom.pa...@gmail.com` on 17 Feb 2011 at 11:39
- Merged into: #334

**Comment by GoogleCodeExporter** [comment]
```
Commenting out the following lines of findNearbyItem() in jquery.flot.js seems 
to fix it:

      // For points and lines, the cursor must be within a
      // certain distance to the data point
   // if (x - mx > maxx || x - mx < -maxx ||
   //     y - my > maxy || y - my < -maxy)
   //     continue;

It seems that this function should work only in terms of pixels as using 
coordinates can be problematic for non-linear axes.

```

Original comment by `tom.pa...@gmail.com` on 18 Feb 2011 at 12:03

**Comment by GoogleCodeExporter** [comment]
```
I found another issue that appears to be the same problem. I'll see if I can 
get this fixed before 0.7. Thanks for the analysis!
```

Original comment by `olau%iol...@gtempaccount.com` on 10 Mar 2011 at 8:08
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
Original comment by `dnsch...@gmail.com` on 7 May 2012 at 11:29
